Buildings/Applications/DHC/CentralPlants/Cooling/Controls/ChillerStage.mo

```diff
@@ -0,0 +1,152 @@
+within Buildings.Applications.DHC.CentralPlants.Cooling.Controls;
+model ChillerStage "Stage controller for chillers"
+  extends Modelica.Blocks.Icons.Block;
+
+  parameter Modelica.SIunits.Time tWai "Waiting time";
+  parameter Modelica.SIunits.Power QEva_nominal
+    "Nominal cooling capaciaty (negative)";
+  parameter Modelica.SIunits.Power  criPoiLoa = 0.55*QEva_nominal
```

Comment:
Are these parameters `cirPoiLoa` and `dQ` allowed to be changed by user? If not, then you should make them `final` or put them in `protected` section.

Reply:
Thanks for your good question. These two parameters are allowed to be changed by the user. More specifically, the thresholds of 0.55 and 0.25 here are default values which allow the users to change.